[Documentation] Include correct dev package

4tXJ7f opened the pull request:
The Ubuntu part of the installation guide mentions `libjson-rpc-cpp-dev`. However, the correct name seems to be `libjsonrpccpp-dev`.

4tXJ7f committed:
[Documentation] Include correct dev package

The Ubuntu part of the installation guide mentions `libjson-rpc-cpp-dev`. However, the correct name seems to be `libjsonrpccpp-dev`.

chriseth commented:
Thanks for the pull request! The name of the package seems to have changed from wily on: http://packages.ubuntu.com/search?keywords=libjsonrpccpp-dev
Could you add that to the PR, please?

4tXJ7f commented:
I'm happy to update the documentation with some additional information. However, I can't seem to find a package called `libjson-rpc-cpp-dev` for older versions of Ubuntu [either](http://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=libjson-rpc-cpp-dev&searchon=names), seems weird (it should show up in the search, no?).

chriseth commented:
Ah, it seems that this is part of our ppa: http://ppa.launchpad.net/ethereum/ethereum/ubuntu/pool/main/libj/libjson-rpc-cpp/ but it only extends until trusty. So for up to vivid, you get it from our ppa, otherwise you get the differently named package from the ofificial ubuntu repository.

4tXJ7f committed:
[Documentation] Add details on libjsonrpccpp-dev

Add explanation to the documentation that the package name changes from  `libjson-rpc-cpp-dev` to  `libjsonrpccpp-dev` and is supplied by the universe repository instead of the Ethereum PPA.

4tXJ7f commented:
Thanks for the info, I updated the doc accordingly.

chriseth commented:
Thanks!
